metaflow/plugins/cards/card_modules/card.py

```diff
@@ -68,8 +94,45 @@ def render(self, task) -> str:
         """
         return NotImplementedError()
 
+    # FIXME document
+    def render_runtime(self, task, data):
+        return
+
+    # FIXME document
+    def refresh(self, task, data):
```

Comment:
structure of `data` is given by the return value of `CardComponentManager._get_latest_data`